CommonWallet/Classes/Common/Commands/WalletPresentationCommand.swift

```diff
@@ -15,13 +15,13 @@ public protocol WalletPresentationCommandProtocol: WalletCommandProtocol {
 }
 
 extension WalletPresentationCommandProtocol {
-    func present(view: ControllerBackedProtocol, in navigation: NavigationProtocol) {
+    func present(view: UIViewController, in navigation: NavigationProtocol) {
         switch presentationStyle {
         case .push(let hidesBottomBar):
-            view.controller.hidesBottomBarWhenPushed = hidesBottomBar
-            navigation.push(view.controller)
+            view.hidesBottomBarWhenPushed = hidesBottomBar
```

Comment:
hideBarOnPush

Reply:
I don't think it will change something significantly